### ISSUE TITLE
Select attributes in Find method - mongodb

### ISSUE THREAD

**Issue by thonythony** [typeorm/typeorm#1929]
**Issue type:**

[ ] question
[ ] bug report
[x] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[x] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `websql`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

According to documentation located in http://typeorm.io/#/find-options, I want to select specific attributes from my find query. I am using MongoDB and this doesn't work.
Can you explain me please ?

Thanks

**Comment by pleerock** [comment]
Some of find options are rdbms specific, can you please provide a code snippet of what you are executing, expected results what you get?

**Comment by thonythony** [comment]
I have an entity with three fields : 
- name
- value
- createdAt
I execute this kind of code : 
`settingRepository.find({ select: ["name"] });`

I expected a list of object composed with only field "name" (so without "value" and "createdAt").
I got a list of object composed with all fields ("name, "value", "createdAt", "id").

I look up very quickly typeorm code and I don't see an implementation of this feature. I look up very quickly mongodb driver javascript code and I don't see an implementation of select feature.
I came from mongoose, and now I think this feature is implemented only by mongoose...
